packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -128,7 +128,7 @@ export class DefaultCrudRepository<T extends Entity, ID>
     this.modelClass = dataSource.createModel<juggler.PersistedModelClass>(
       definition.name,
       properties,
-      Object.assign({strict: true}, definition.settings),
+      Object.assign({strict: true}, {strictDelete: true}, definition.settings),
```

Comment:
It's simpler to have `Object.assign({strict: true, strictDelete: true}, definition.settings),`.

Reply:
Ah, indeed.

Reply:
I think we probably should add an optional `options` argument  the constructor so that these flags can be customized.